meta-mentor-staging/recipes-graphics/mesa/mesa-demos_8.2.0.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend_amd := "${THISDIR}/${PN}_${PV}:"
+
+SRC_URI_append_amd = "\
```

Comment:
Does this bug not exist for non-amd setups? If it's a general bug, this override usage is inappropriate here.